### ISSUE TITLE
Issue running DMB within Portainer in a Proxmox LXC

### ISSUE THREAD

**Issue by profucius** [I-am-PUID-0/DMB#50]
**Please complete the following information:**
 - OS: Proxmox LXC with only Portainer/Docker.
 - Version: Portainer 2.21.0 Community Edition (self-hosted)
 - Supporting Applications: 

**Describe the bug**

I am trying to install DMB into Portainer within Proxmox. Everything is freshly installed, Proxmox and Portainer. I get the error below when I try to deploy DMB in a new Portainer stack.

```
Failed to deploy a stack: time="2024-09-10T15:19:22Z" level=warning msg="/data/compose/7/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion" 
Container DMB Recreate Container DMB Recreated Container DMB Starting 
Error response from daemon: error gathering device information while adding custom device "/dev/fuse": no such file or directory
```

I am assuming this is important, so I shouldn't keep it disabled. However I am a docker novice (which is why I'm using Proxmox and Portainer to make the process easier) so I am not sure what to do from here.


**To Reproduce**

I open Portainer and go into Stacks. I add a Stack and use the Web editor to paste the docker compose yaml from your Readme.md. I lightly edit it to include my RD_API_KEY and PLEX_TOKEN/ADDRESS. I then Deploy the Stack, and moments later I get the error above.

If I comment-out the `Devices` and it's following line, then the Stack will start. However then I get a new issue:

```
INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
```

Below I have pasted my docker compose yaml, and the whole container log (with the devices header disabled). Perhaps it will help elucidate something I am doing wrong?

### Docker compose yaml I am using:
```
version: "3.8"

services:
  DMB:
    container_name: DMB
    image: iampuid0/dmb:latest
    stop_grace_period: 60s
    stdin_open: true
    tty: true
    volumes:
      - /home/username/docker/DMB/config:/config
      - /home/username/docker/DMB/log:/log
      - /home/username/docker/DMB/Zurg/RD:/zurg/RD
      - /home/username/docker/DMB/Zurg/AD:/zurg/AD
      - /home/username/docker/DMB/Zurg/mnt:/data:shared
      - /home/username/docker/DMB/Riven/data:/riven/backend/data
      - /home/username/docker/DMB/Riven/mnt:/mnt
      - /home/username/docker/DMB/PostgreSQL/data:/postgres_data
      - /home/username/docker/DMB/Zilean/data:/zilean/app/data
    environment:
      - TZ=
      - PUID=
      - PGID=
      - ZURG_ENABLED=true
      - RD_API_KEY=xxxxxx
      - RCLONE_MOUNT_NAME=DMB
      - RIVEN_BACKEND_ENABLED=true
      - RIVEN_FRONTEND_ENABLED=true
      - ORIGIN=http://0.0.0.0:3000
      - PLEX_TOKEN=xxxxxx
      - PLEX_ADDRESS=http://xxxxxx:32400
    ports:
      - "3000:3000"
    # devices:
     # - /dev/fuse:/dev/fuse:rwm
    cap_add:
      - SYS_ADMIN
    security_opt:
      - apparmor:unconfined
      - no-new-privileges
```

### Portainer output logs (assuming I disabled Devices header so it would actually start):
```
Sep 10, 2024 15:32:07 - INFO - 
                                                                       
DDDDDDDDDDDDD        MMMMMMMM               MMMMMMMMBBBBBBBBBBBBBBBBB   
D::::::::::::DDD     M:::::::M             M:::::::MB::::::::::::::::B  
D:::::::::::::::DD   M::::::::M           M::::::::MB::::::BBBBBB:::::B 
DDD:::::DDDDD:::::D  M:::::::::M         M:::::::::MBB:::::B     B:::::B
  D:::::D    D:::::D M::::::::::M       M::::::::::M  B::::B     B:::::B
  D:::::D     D:::::DM:::::::::::M     M:::::::::::M  B::::B     B:::::B
  D:::::D     D:::::DM:::::::M::::M   M::::M:::::::M  B::::BBBBBB:::::B 
  D:::::D     D:::::DM::::::M M::::M M::::M M::::::M  B:::::::::::::BB  
  D:::::D     D:::::DM::::::M  M::::M::::M  M::::::M  B::::BBBBBB:::::B 
  D:::::D     D:::::DM::::::M   M:::::::M   M::::::M  B::::B     B:::::B
  D:::::D     D:::::DM::::::M    M:::::M    M::::::M  B::::B     B:::::B
  D:::::D    D:::::D M::::::M     MMMMM     M::::::M  B::::B     B:::::B
DDD:::::DDDDD:::::D  M::::::M               M::::::MBB:::::BBBBBB::::::B
D:::::::::::::::DD   M::::::M               M::::::MB:::::::::::::::::B 
D::::::::::::DDD     M::::::M               M::::::MB::::::::::::::::B  
DDDDDDDDDDDDD        MMMMMMMM               MMMMMMMMBBBBBBBBBBBBBBBBB   
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       

                              Version: 5.1.0                                    



Sep 10, 2024 15:32:07 - INFO - Group with GID 1001 does not exist. Creating group...
Sep 10, 2024 15:32:07 - INFO - User 'DMB' does not exist. Creating user...
Sep 10, 2024 15:32:33 - INFO - Created system user 'DMB' with UID 1001 and GID 1001.
Sep 10, 2024 15:32:33 - INFO - Setting up Zurg
Sep 10, 2024 15:32:33 - INFO - Setting up Zurg w/ RealDebrid instance in directory: /zurg/RD/
Sep 10, 2024 15:32:33 - INFO - Found Zurg version v0.9.3-final in /zurg/RD/
Sep 10, 2024 15:32:33 - INFO - Preparing Zurg instance for RealDebrid
Sep 10, 2024 15:32:33 - INFO - Using Zurg app found for RealDebrid in /zurg/RD/
Sep 10, 2024 15:32:33 - INFO - Zurg setup process complete
Sep 10, 2024 15:32:33 - INFO - Automatic update disabled for Zurg
Sep 10, 2024 15:32:33 - INFO - Starting Zurg w/ RealDebrid subprocess
Sep 10, 2024 15:32:33 - INFO - Zurg process started with PID: 14
Sep 10, 2024 15:32:33 - INFO - Checking rclone flags
Sep 10, 2024 15:32:33 - INFO - Configuring the rclone mount name to "DMB"
Sep 10, 2024 15:32:33 - INFO - Zurg w/ RealDebrid subprocess: Logging to logs/zurg-2024-09-10.log
Sep 10, 2024 15:32:33 - INFO - Configuring rclone for DMB
Sep 10, 2024 15:32:33 - INFO - Zurg w/ RealDebrid subprocess: 	zurg	Version: v0.9.3-final
Sep 10, 2024 15:32:33 - INFO - Zurg w/ RealDebrid subprocess: 	zurg	GitCommit: 4179c2745b4fb22fcb37f36de27b3daa39f114f0
Sep 10, 2024 15:32:33 - INFO - Zurg w/ RealDebrid subprocess: 	zurg	BuiltAt: 2024-07-14T09:48:30
Sep 10, 2024 15:32:33 - INFO - Zurg w/ RealDebrid subprocess: 	zurg	Debug logging is enabled; if you are not debugging please set LOG_LEVEL=info in your environment
Sep 10, 2024 15:32:33 - INFO - Zurg w/ RealDebrid subprocess: 	zurg	Starting server on [::]:9452
Sep 10, 2024 15:32:34 - INFO - Zurg w/ RealDebrid subprocess: 	zurg	Your account will expire in 69 days
Sep 10, 2024 15:32:38 - INFO - Waiting to start the rclone process until the Zurg WebDAV http://localhost:9452/dav/ is accessible.
Sep 10, 2024 15:32:38 - INFO - The Zurg WebDAV URL http://localhost:9452/dav is accessible. Starting rclone for DMB
Sep 10, 2024 15:32:38 - INFO - Starting rclone w/ DMB subprocess
Sep 10, 2024 15:32:38 - INFO - rclone process started with PID: 23
Sep 10, 2024 15:32:38 - INFO - rclone startup complete
Sep 10, 2024 15:32:38 - INFO - Setting up PostgreSQL...
Sep 10, 2024 15:32:38 - INFO - PostgreSQL data directory exists at /postgres_data.
Sep 10, 2024 15:32:38 - INFO - Changed ownership and set permissions of /postgres_data.
Sep 10, 2024 15:32:38 - INFO - Starting PostgreSQL subprocess
Sep 10, 2024 15:32:38 - INFO - PostgreSQL process started with PID: 33
Sep 10, 2024 15:32:38 - INFO - Checking if PostgreSQL server has started...
Sep 10, 2024 15:32:38 - INFO - PostgreSQL subprocess: 2024-09-10 15:32:38.457 UTC [33] LOG:  starting PostgreSQL 16.3 on x86_64-alpine-linux-musl, compiled by gcc (Alpine 13.2.1_git20240309) 13.2.1 20240309, 64-bit
Sep 10, 2024 15:32:38 - INFO - PostgreSQL subprocess: 2024-09-10 15:32:38.457 UTC [33] LOG:  listening on IPv6 address "::1", port 5432
Sep 10, 2024 15:32:38 - INFO - PostgreSQL subprocess: 2024-09-10 15:32:38.457 UTC [33] LOG:  listening on IPv4 address "127.0.0.1", port 5432
Sep 10, 2024 15:32:38 - INFO - PostgreSQL subprocess: 2024-09-10 15:32:38.461 UTC [33] LOG:  listening on Unix socket "/run/postgresql/.s.PGSQL.5432"
Sep 10, 2024 15:32:38 - INFO - PostgreSQL subprocess: 2024-09-10 15:32:38.467 UTC [40] LOG:  database system was shut down at 2024-09-10 15:19:22 UTC
Sep 10, 2024 15:32:38 - INFO - rclone mount name "DMB": 2024/09/10 15:32:38 mount helper error: fusermount3: fuse device not found, try 'modprobe fuse' first
Sep 10, 2024 15:32:38 - INFO - rclone mount name "DMB": 2024/09/10 15:32:38 Fatal error: failed to mount FUSE fs: fusermount: exit status 1
Sep 10, 2024 15:32:38 - INFO - PostgreSQL subprocess: 2024-09-10 15:32:38.475 UTC [33] LOG:  database system is ready to accept connections
Sep 10, 2024 15:32:40 - INFO - PostgreSQL subprocess: 2024-09-10 15:32:40.449 UTC [47] FATAL:  role "root" does not exist
Sep 10, 2024 15:32:40 - INFO - PostgreSQL server has started.
Sep 10, 2024 15:32:40 - INFO - Database 'riven' already exists.
Sep 10, 2024 15:32:40 - INFO - Checking if PostgreSQL database is accessible...
Sep 10, 2024 15:32:40 - INFO - PostgreSQL database is accessible.
Sep 10, 2024 15:32:40 - INFO - Listing PostgreSQL database sizes...
Sep 10, 2024 15:32:40 - INFO - Database: postgres, Size: 7508 kB
Sep 10, 2024 15:32:40 - INFO - Database: riven, Size: 7508 kB
Sep 10, 2024 15:32:40 - INFO - Database: template1, Size: 7417 kB
Sep 10, 2024 15:32:40 - INFO - Database: template0, Size: 7353 kB
Sep 10, 2024 15:32:40 - INFO - Configuring riven_backend
Sep 10, 2024 15:32:40 - INFO - Fetching latest riven release.
Sep 10, 2024 15:32:40 - INFO - riven latest release: v0.11.1
Sep 10, 2024 15:32:40 - INFO - Fetching release information from https://api.github.com/repos/rivenmedia/riven/releases/tags/v0.11.1
Sep 10, 2024 15:32:40 - INFO - No assets found. Using zipball_url.
Sep 10, 2024 15:32:40 - INFO - Downloading from https://api.github.com/repos/rivenmedia/riven/zipball/v0.11.1
Sep 10, 2024 15:32:41 - INFO - Successfully downloaded the latest release for riven_backend
Sep 10, 2024 15:32:41 - INFO - Automatic update disabled for riven_backend
Sep 10, 2024 15:32:41 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 15:32:46 - INFO - Zurg w/ RealDebrid subprocess: 	manager	Fetched info for 1114 torrents
Sep 10, 2024 15:32:46 - INFO - Zurg w/ RealDebrid subprocess: 	manager	Compiled into 1113 torrents, 0 were missing info
Sep 10, 2024 15:32:47 - INFO - Zurg w/ RealDebrid subprocess: 	manager	Starting periodic repair job
Sep 10, 2024 15:32:47 - INFO - Zurg w/ RealDebrid subprocess: 	manager	Periodic repair started; searching for broken torrents
Sep 10, 2024 15:32:47 - INFO - Zurg w/ RealDebrid subprocess: 	manager	Starting periodic refresh job
Sep 10, 2024 15:32:47 - INFO - Zurg w/ RealDebrid subprocess: 	manager	Finished repairing 0 broken torrents
Sep 10, 2024 15:32:51 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 15:33:01 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 15:33:11 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 15:33:18 - ERROR - The Riven frontend process is not running. | The Riven backend process is not running. | The Rclone RD process for DMB is not running. | The Zilean process is not running.
Sep 10, 2024 15:33:21 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 15:33:31 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 15:33:41 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
```

**Comment by profucius** [comment]
That helped me so much, thank you! I re-enabled the Devices in the yaml and then recomposed, and started the stack. It successfully started and I was able to access the Riven WebUI.

Looking at the logs, I do see one more error, not sure if this is something related or new?
```
Sep 10, 2024 16:20:02 - ERROR - The Zilean process is not running.
```

**Comment by profucius** [comment]
Ack, something happened and I'm getting a new issue. It may be related so I'm posted it in this same thread.

I broke the container/stack, so I decided to start all over. I deleted and recreated the entire LXC for Portainer. I made sure to enable FUSE again before proceeding, as well as running the two commands you mentioned in the "Define the directory structure". I then used the docker compose yaml from earlier and created the stack/container. I ran it, and this is what I am seeing in the logs. I cannot access Riven from the WebUI.
```
Sep 10, 2024 20:07:27 - INFO - 
                                                                       
DDDDDDDDDDDDD        MMMMMMMM               MMMMMMMMBBBBBBBBBBBBBBBBB   
D::::::::::::DDD     M:::::::M             M:::::::MB::::::::::::::::B  
D:::::::::::::::DD   M::::::::M           M::::::::MB::::::BBBBBB:::::B 
DDD:::::DDDDD:::::D  M:::::::::M         M:::::::::MBB:::::B     B:::::B
  D:::::D    D:::::D M::::::::::M       M::::::::::M  B::::B     B:::::B
  D:::::D     D:::::DM:::::::::::M     M:::::::::::M  B::::B     B:::::B
  D:::::D     D:::::DM:::::::M::::M   M::::M:::::::M  B::::BBBBBB:::::B 
  D:::::D     D:::::DM::::::M M::::M M::::M M::::::M  B:::::::::::::BB  
  D:::::D     D:::::DM::::::M  M::::M::::M  M::::::M  B::::BBBBBB:::::B 
  D:::::D     D:::::DM::::::M   M:::::::M   M::::::M  B::::B     B:::::B
  D:::::D     D:::::DM::::::M    M:::::M    M::::::M  B::::B     B:::::B
  D:::::D    D:::::D M::::::M     MMMMM     M::::::M  B::::B     B:::::B
DDD:::::DDDDD:::::D  M::::::M               M::::::MBB:::::BBBBBB::::::B
D:::::::::::::::DD   M::::::M               M::::::MB:::::::::::::::::B 
D::::::::::::DDD     M::::::M               M::::::MB::::::::::::::::B  
DDDDDDDDDDDDD        MMMMMMMM               MMMMMMMMBBBBBBBBBBBBBBBBB   
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       

                              Version: 5.1.2                                    



Sep 10, 2024 20:07:27 - INFO - Group with GID 1001 does not exist. Creating group...
Sep 10, 2024 20:07:27 - INFO - User 'DMB' does not exist. Creating user...
Sep 10, 2024 20:07:48 - INFO - Created system user 'DMB' with UID 1001 and GID 1001.
Sep 10, 2024 20:07:48 - ERROR - Please set the debrid API Key: environment variable is missing from the docker-compose file
Sep 10, 2024 20:07:48 - INFO - Setting up PostgreSQL...
Sep 10, 2024 20:07:48 - INFO - PostgreSQL data directory exists at /postgres_data.
Sep 10, 2024 20:07:48 - INFO - Changed ownership and set permissions of /postgres_data.
Sep 10, 2024 20:07:48 - INFO - Initializing PostgreSQL data directory at /postgres_data...
Sep 10, 2024 20:07:48 - INFO - Starting PostgreSQL_init subprocess
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init process started with PID: 11
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: The files belonging to this database system will be owned by user "DMB".
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: This user must also own the server process.
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: The database cluster will be initialized with this locale configuration:
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: provider:    libc
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: LC_COLLATE:  C
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: LC_CTYPE:    C.UTF-8
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: LC_MESSAGES: C
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: LC_MONETARY: C
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: LC_NUMERIC:  C
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: LC_TIME:     C
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: The default database encoding has accordingly been set to "UTF8".
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: The default text search configuration will be set to "english".
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: Data page checksums are disabled.
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: fixing permissions on existing directory /postgres_data ... ok
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: creating subdirectories ... ok
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: selecting dynamic shared memory implementation ... posix
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: selecting default max_connections ... 100
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: selecting default shared_buffers ... 128MB
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: selecting default time zone ... UTC
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: creating configuration files ... ok
Sep 10, 2024 20:07:48 - INFO - PostgreSQL_init subprocess: running bootstrap script ... ok
Sep 10, 2024 20:07:49 - INFO - PostgreSQL_init subprocess: performing post-bootstrap initialization ... ok
Sep 10, 2024 20:07:49 - INFO - PostgreSQL_init subprocess: initdb: warning: enabling "trust" authentication for local connections
Sep 10, 2024 20:07:49 - INFO - PostgreSQL_init subprocess: syncing data to disk ... ok
Sep 10, 2024 20:07:49 - INFO - PostgreSQL_init subprocess: Success.
Sep 10, 2024 20:07:49 - INFO - PostgreSQL_init subprocess: initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
Sep 10, 2024 20:07:49 - INFO - Initialized PostgreSQL data directory at /postgres_data.
Sep 10, 2024 20:07:49 - INFO - Starting PostgreSQL subprocess
Sep 10, 2024 20:07:49 - INFO - PostgreSQL process started with PID: 22
Sep 10, 2024 20:07:49 - INFO - Checking if PostgreSQL server has started...
Sep 10, 2024 20:07:49 - INFO - PostgreSQL subprocess: 2024-09-10 20:07:49.312 UTC [22] LOG:  starting PostgreSQL 16.3 on x86_64-alpine-linux-musl, compiled by gcc (Alpine 13.2.1_git20240309) 13.2.1 20240309, 64-bit
Sep 10, 2024 20:07:49 - INFO - PostgreSQL subprocess: 2024-09-10 20:07:49.312 UTC [22] LOG:  listening on IPv6 address "::1", port 5432
Sep 10, 2024 20:07:49 - INFO - PostgreSQL subprocess: 2024-09-10 20:07:49.312 UTC [22] LOG:  listening on IPv4 address "127.0.0.1", port 5432
Sep 10, 2024 20:07:49 - INFO - PostgreSQL subprocess: 2024-09-10 20:07:49.315 UTC [22] LOG:  listening on Unix socket "/run/postgresql/.s.PGSQL.5432"
Sep 10, 2024 20:07:49 - INFO - PostgreSQL subprocess: 2024-09-10 20:07:49.318 UTC [27] LOG:  database system was shut down at 2024-09-10 20:07:49 UTC
Sep 10, 2024 20:07:49 - INFO - PostgreSQL subprocess: 2024-09-10 20:07:49.323 UTC [22] LOG:  database system is ready to accept connections
Sep 10, 2024 20:07:51 - INFO - PostgreSQL subprocess: 2024-09-10 20:07:51.299 UTC [32] FATAL:  role "root" does not exist
Sep 10, 2024 20:07:51 - INFO - PostgreSQL server has started.
Sep 10, 2024 20:07:51 - INFO - Database 'riven' created successfully with owner 'DMB'.
Sep 10, 2024 20:07:51 - INFO - Checking if PostgreSQL database is accessible...
Sep 10, 2024 20:07:51 - INFO - PostgreSQL database is accessible.
Sep 10, 2024 20:07:51 - INFO - Listing PostgreSQL database sizes...
Sep 10, 2024 20:07:51 - INFO - Database: postgres, Size: 7508 kB
Sep 10, 2024 20:07:51 - INFO - Database: riven, Size: 7508 kB
Sep 10, 2024 20:07:51 - INFO - Database: template1, Size: 7353 kB
Sep 10, 2024 20:07:51 - INFO - Database: template0, Size: 7353 kB
Sep 10, 2024 20:07:51 - INFO - Configuring riven_backend
Sep 10, 2024 20:07:51 - INFO - Fetching latest riven release.
Sep 10, 2024 20:07:51 - INFO - riven latest release: v0.11.1
Sep 10, 2024 20:07:51 - INFO - Fetching release information from https://api.github.com/repos/rivenmedia/riven/releases/tags/v0.11.1
Sep 10, 2024 20:07:51 - INFO - No assets found. Using zipball_url.
Sep 10, 2024 20:07:51 - INFO - Downloading from https://api.github.com/repos/rivenmedia/riven/zipball/v0.11.1
Sep 10, 2024 20:07:52 - INFO - Successfully downloaded the latest release for riven_backend
Sep 10, 2024 20:07:52 - INFO - Automatic update disabled for riven_backend
Sep 10, 2024 20:07:52 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 20:08:02 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 20:08:12 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 20:08:22 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 20:08:32 - INFO - Waiting for symlink directory /data/DMB/__all__ to become available before starting riven_backend
Sep 10, 2024 20:08:37 - ERROR - The Riven frontend process is not running. | The Riven backend process is not running.
```
It just keeps looping through those last few lines over and over again. I am not sure why, I do have FUSE enabled and the DMB directory created. Can I check something else?

Also oddly, if I go to Stop the stack in Portainer, these lines will show in the logs. I can then access Riven via WebUI but I only get a 500 Internal Error. Strange that the frontenbecomes available only after Stopping.

```
Sep 10, 2024 20:14:18 - INFO - Shutdown signal received. Cleaning up...
Sep 10, 2024 20:14:18 - INFO - Stopping riven_frontend
Sep 10, 2024 20:14:18 - WARNING - riven_frontend process not found or already stopped.
Sep 10, 2024 20:14:18 - INFO - Stopping riven_backend
Sep 10, 2024 20:14:18 - WARNING - riven_backend process not found or already stopped.
Sep 10, 2024 20:14:18 - INFO - Stopping Zilean
Sep 10, 2024 20:14:18 - WARNING - Zilean process not found or already stopped.
Sep 10, 2024 20:14:18 - INFO - Stopping PostgreSQL
Sep 10, 2024 20:14:18 - INFO - PostgreSQL subprocess: 2024-09-10 20:14:18.247 UTC [22] LOG:  received smart shutdown request
Sep 10, 2024 20:14:18 - INFO - PostgreSQL subprocess: 2024-09-10 20:14:18.253 UTC [22] LOG:  background worker "logical replication launcher" (PID 30) exited with exit code 1
Sep 10, 2024 20:14:18 - INFO - PostgreSQL subprocess: 2024-09-10 20:14:18.388 UTC [25] LOG:  checkpoint complete: wrote 966 buffers (5.9%); 0 WAL file(s) added, 0 removed, 0 recycled; write=88.817 s, sync=0.127 s, total=88.970 s; sync files=310, longest=0.005 s, average=0.001 s; distance=4512 kB, estimate=4512 kB; lsn=0/1953170, redo lsn=0/1953138
Sep 10, 2024 20:14:18 - INFO - PostgreSQL subprocess: 2024-09-10 20:14:18.388 UTC [25] LOG:  shutting down
Sep 10, 2024 20:14:18 - INFO - PostgreSQL subprocess: 2024-09-10 20:14:18.392 UTC [25] LOG:  checkpoint starting: shutdown immediate
Sep 10, 2024 20:14:18 - INFO - PostgreSQL subprocess: 2024-09-10 20:14:18.410 UTC [25] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.001 s, sync=0.001 s, total=0.022 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=4060 kB; lsn=0/19531E8, redo lsn=0/19531E8
Sep 10, 2024 20:14:18 - INFO - PostgreSQL subprocess: 2024-09-10 20:14:18.416 UTC [22] LOG:  database system is shut down
Sep 10, 2024 20:14:18 - INFO - PostgreSQL process terminated gracefully.
Sep 10, 2024 20:14:18 - INFO - Stopping Zurg
Sep 10, 2024 20:14:18 - WARNING - Zurg process not found or already stopped.
Sep 10, 2024 20:14:18 - INFO - Stopping rclone
Sep 10, 2024 20:14:18 - WARNING - rclone process not found or already stopped.
Sep 10, 2024 20:14:18 - INFO - Shutdown complete.
Sep 10, 2024 20:14:18 - INFO - Configuring riven_frontend
Sep 10, 2024 20:14:18 - INFO - Fetching latest riven-frontend release.
Sep 10, 2024 20:14:18 - INFO - riven-frontend latest release: v0.11.2
Sep 10, 2024 20:14:18 - INFO - riven_frontend current version: v0.11.2
Sep 10, 2024 20:14:18 - INFO - Automatic update disabled for riven_frontend
Sep 10, 2024 20:14:18 - INFO - Starting riven_frontend subprocess
Sep 10, 2024 20:14:18 - INFO - riven_frontend process started with PID: 95
Sep 10, 2024 20:14:19 - INFO - riven_frontend subprocess: Listening on 0.0.0.0:3000
```

**Comment by profucius** [comment]
Facepalm, yep that fixed it haha. I was juggling too many changes at once and didn't realize I had disabled the token or that doing so would cause the rest to stop. I'm back on track now. Trying Zilean next!

Since I'm here typing, is it normal to see these lines at the start of the logs every restart? If so, I won't concern about it.
```
Sep 10, 2024 15:26:01 - INFO - Group with GID 1001 does not exist. Creating group...
Sep 10, 2024 15:26:02 - INFO - User 'DMB' does not exist. Creating user...
```

**Comment by profucius** [comment]
Great. Thank you again for your fast and helpful responses. It looks like things are working well and this issue is resolved. I'll close it out now.

For anyone who finds this looking for a solution, the key points are:

- Ensure that the Proxmox LXC has FUSE set to enabled within Options > Features.
- Ensure you've created the directories according to the DMB wiki instructions.
- Ensure you have a Real Debrid token in the docker compose yaml.
